gui/autobutcher.lua

```diff
@@ -306,6 +306,13 @@ function WatchList:init()
                         frame={t=1, l=82},
                         text='ordered'
                     },
+                    widgets.Label{
+                        view_id='disabled_warning',
+                        visible=not plugin.isEnabled,
```

Comment:
```suggestion
                        visible=function() return not plugin.isEnabled() end,
```

Reply:
you can add a similar (but inverted) `visible` attribute to the list; then you can remove the code added to `refresh()`

the list one can just look like this:
```lua
visible=plugin.isEnabled
```
since it's already a function with the correct signature